src/config.rs

```diff
@@ -62,6 +62,7 @@ const RAFT_MAX_MEM: usize = 2 * GB as usize;
 const LAST_CONFIG_FILE: &str = "last_tikv.toml";
 const TMP_CONFIG_FILE: &str = "tmp_tikv.toml";
 const MAX_BLOCK_SIZE: usize = 32 * MB as usize;
+const DEFAULT_REFILL_PERIOD_US: i64 = 100 * 1000; // 100ms
```

Comment:
Can you rebase after https://github.com/tikv/tikv/pull/8371 is merged?

Reply:
rebased